docs/syntax.md

```diff
@@ -1093,7 +1093,7 @@ Result/wrap x = (Result/Ok x)
 with Result {
   ask x = (some_operation ...)
   ask y = (some_operation ...)
-  wrap(x * y)
+  (wrap (* x y)
```

Comment:
It's missing a second parentheses after the last one
```
(wrap (* x y))
```

Reply:
Dang, I could swear I double-checked the change 😅  Thanks!